docs/UserGuide.md

```diff
@@ -16,7 +16,55 @@ CS students are incredibly busy and hence having a command line interface (CLI)
 1. Ensure that you have Java 11 or above installed.
 1. Down the latest version of `DukeOfBooks` from [here](https://github.com/AY2223S2-CS2113-F10-4/tp).
 
-## Features 
+## Access Management
+The application requires user to login before performing other actions.
+
+### Login: `login`
+Login the system using username and password. 
+
+Format:
+* login -username [username] -password [password]
+
+Examples:
+* login -username me -password my$password
+
+### Signup: `signup`
+Sign up a new account with a unique username, password and user's actual name.
+
+Format: 
+* signup -username [username] -password [password] -name [your name]
+
+Examples:
+* signup -username me -password my$password -name My Name
+
+### Change password: `password`
+Change the password of a user.
+
+Format: 
+* password -username [username] -old [old password] -new [new password]
+
+Examples:
+* password -username me -old old$password -new new$password
+
+### Logout: `logout`
+Logout the user. This command can only be used after logging in.
+
```

Comment:
Would be good to explain in detail or show an output of this command for better clarity